v1.0/multi-active-availability.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Multi-Active Availability
+summary: Learn about CockroachDB's high availability model, known as Multi-Active Availability
+toc: false
+---
+
+CockroachDB's availability model is described as "Multi-Active Availability." In essence, multi-active availability provides benefits similar to traditional notions of high availability, but also lets you read and write from every node in your cluster without generating any conflicts.
```

Comment:
Might want to link to the "What is Multi-Active Availability" section here.

Reply:
Given that the TOC is right beneath this and the information is sequential, will leave out link but can add it in later if we get more feedback that users expect it.